man/fmidi-read.1

```diff
@@ -0,0 +1,27 @@
+.TH FMIDI\-READ "1" "August 2021" "" "User Commands"
+.SH NAME
+fmidi\-read \(en validate MIDI files
```

Comment:
> validate MIDI files

This is not the role of this program. It's permissive file reading, just like indicated in the readme.
The goal is to print a textual s-exp representation of a music file.
It will only return a failure code if the file is broken beyond repair.

There may be a `fmidi-lint` at a future time, which actually reports MIDI files which don't conform well.